website/pages/docs/plugins/destinations/mongodb/overview.md

```diff
@@ -0,0 +1,52 @@
+# MongoDB Destination Plugin
+
+import { getLatestVersion } from "../../../../../utils/versions";
+import { Badge } from "../../../../../components/Badge";
+import { Callout } from 'nextra-theme-docs'
+
+<Badge text={"Latest: " + getLatestVersion("destination", "mongodb")}/>
+
+This destination plugin lets you sync data from a CloudQuery source to a MongoDB.
+
+Supported database versions:
+
+- MongoDB >= 3.6 (The same minimum version supported by the official [Go driver](https://github.com/mongodb/mongo-go-driver))
+
+## Configuration
+
+### Example
+
+This example configures a MongoDB destination, located at `localhost:27017`. The (top level) spec section is described in the [Destination Spec Reference](/docs/reference/destination-spec).
+
+```yaml
+kind: destination
+spec:
+  name: "mongodb"
+  registry: "github"
+  path: "cloudquery/mongodb"
+  version: "VERSION_DESTINATION_MONGODB"
+  spec:
+    connection_string: "mongodb://localhost:27017"
+    database: "your_mongo_database_name"
+```
+
+<Callout type="info">
+Make sure you use environment variable expansion in production instead of committing the credentials to the configuration file directly.
```

Comment:
```suggestion
Make sure to use [environment variable substitution](/docs/advanced-topics/environment-variable-substitution) in production instead of committing the credentials to the configuration file directly.
```